x-pack/plugins/index_lifecycle_management/common/constants/user_action.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const UA_APP_NAME = 'index_lifecycle_management';
+
+export const UA_APP_LOAD = 'app_load';
+export const UA_POLICY_CREATE = 'policy_create';
+export const UA_POLICY_UPDATE = 'policy_update';
+export const UA_POLICY_DELETE = 'policy_delete';
+export const UA_POLICY_ATTACH_INDEX = 'policy_attach_index';
+export const UA_POLICY_ATTACH_INDEX_TEMPLATE = 'policy_attach_index_template';
+export const UA_POLICY_DETACH_INDEX = 'policy_detach_index';
+export const UA_POLICY_EDIT_CLICK = 'policy_edit_click';
```

Comment:
I don't see where this action is being tracked. There is a `UA_EDIT_CLICK` being tracked but not this one.

Reply:
You have eagle eyes!